Cohort2/team4/server/src/routers/user.ts

```diff
@@ -0,0 +1,39 @@
+import { t } from '..//app';
```

Comment:
This isn't going to work. "t" is not exported from the app.js file. So this import will fail.

Let's setup a trpc file that we can import from. I'll put how to do that in another comment. Once it is in place, you can replace this line with: `import { router, publicProcedure } from '../trpc';`